docs-src/concepts/what-is-a-workflow-definition.md

```diff
@@ -8,15 +8,136 @@ tags:
   - explanation
 ---
 
-A Workflow Definition is the code that defines the constraints of a Workflow Execution.
+A Workflow Definition is the code that defines the Workflow and written with a programming language and a corresponding Temporal SDK.
+A Workflow Definition defines the constraints of a Workflow Execution.
+
```

Comment:
I would add something like, "It is the primary function or object method, depending on the language, that defines the end-to-end series of steps of a Temporal Application".